src/query/processor/cypher/runtime/OperatorExecutor.cpp

```diff
@@ -219,16 +426,28 @@ void OperatorExecutor::Filter(SharedBuffer &buffer, std::string jsonPlan, GraphC
     std::thread result(method, std::ref(*this), std::ref(sharedBuffer), query["NextOperator"], gc);
 
     auto condition = query["condition"];
-    FilterHelper FilterHelper(condition.dump());
+    FilterHelper filterHelper(condition.dump());
+
+    std::vector<std::string> batch;
+    batch.reserve(100);
```

Comment:
The value 100 can be converted to a meaningful constant and can be used in this code segment